src/Qowaiv.CodeAnalysis.CSharp/Rule.cs

```diff
@@ -42,14 +42,15 @@ public static partial class Rule
         title: "Characters with Trojan Horse potential are not allowed",
         message: "Trojan Horse character U+{0:X} detected.",
         description:
-            "When char",
+            "When characters with Trojan Horse potential are found, they should be blocked " +
```

Comment:
@Corniel Please check if this is what you wanted to convey

Reply:
Yep.